### ISSUE TITLE
Unable to resolve "AccessibilityInfo" from "node_modules/@ramotion/react-native-circle-menu/node_modules/react-native/Libraries/react-native/react-native-implementation.js"

### ISSUE THREAD

**Issue by criticalmiind** [Ramotion/react-native-circle-menu#16]
can anyone help me to resolve this issue please.?

```Unable to resolve "AccessibilityInfo" from "node_modules/@ramotion/react-native-circle-menu/node_modules/react-native/Libraries/react-native/react-native-implementation.js"
Failed building JavaScript bundle.
Unable to resolve "ActivityIndicator" from "node_modules/@ramotion/react-native-circle-menu/node_modules/react-native/Libraries/react-native/react-native-implementation.js"
Failed building JavaScript bundle.
Unable to resolve "AccessibilityInfo" from "node_modules/@ramotion/react-native-circle-menu/node_modules/react-native/Libraries/react-native/react-native-implementation.js"
Failed building JavaScript bundle.
Unable to resolve "AccessibilityInfo" from "node_modules/@ramotion/react-native-circle-menu/node_modules/react-native/Libraries/react-native/react-native-implementation.js"
Failed building JavaScript bundle.```

**Comment by IamNotHuman** [comment]
Same error here!

**Comment by oussemajj** [comment]
Same error Here!!

**Comment by Egi-B** [comment]
Same here. Using expo sdk-37

**Comment by masymars** [comment]
same here

**Comment by JosePatricio** [comment]
anyone solved this ?

**Comment by murilokrugner** [comment]
same here!!!!

**Comment by DimarDM** [comment]
same here

**Comment by iliassanati** [comment]
same here

**Comment by ansarikhurshid786** [comment]
same error

**Comment by ansarikhurshid786** [comment]
anyone got solution?

**Comment by ansarikhurshid786** [comment]
i found that this project has react native version 0.48.4 so if u upgrade to 55 it will works.

**Comment by ansarikhurshid786** [comment]
https://stackoverflow.com/questions/51414510/react-native-build-error-accessibilityinfo-does-not-exist
